schemas/cluster_version_mapping.schema.json

```diff
@@ -0,0 +1,34 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "$id": "cluster_version_mapping.schema.json",
+    "title": "Cluster Version Mapping for Rapid Recommendations",
+    "descriptions": "A data structure used by the Insights Operator Gathering Conditions Service v2 API to map Insights Operator versions to specific remote configurations for those versions.",
+    "examples": [
+        [
+            ["1.0.0", "safe_1.json"],
+            ["2.0.0-0", "experimental_1.json"],
+            ["2.0.0", "safe_2.json"],
+            ["3.0.0-0", "experimental_2.json"]
+        ]
+    ],
+    "type": "array",
+    "description": "A sequence of pairs specifying OpenShift version intervals and specific remote configurations to be used for those intervals. The sequence of pairs in the array must specify a strictly increasing sequence of versions (this requirement is not imposed by the schema itself). An interval is defined by two consecutive pairs.",
+    "uniqueItems": true,
+    "items": {
+        "type": "array",
+        "description": "A pair specifying the start of an OpenShift version interval and a specific remote configuration to be used for a version interval that starts at the version (inclusive) and ends at the version specified by the next pair (non-inclusive). The interval specified by the last pair is unlimited",
```

Comment:
I would have used `unbound` rather than `unlimited`, but to be honest, I have not idea what is the most correct word in English.

Reply:
Thanks. Changed.

It seems that both terms are used these days, but I think that "unbounded" is better.